cli/command/network/create.go

```diff
@@ -51,6 +53,12 @@ func newCreateCommand(dockerCli command.Cli) *cobra.Command {
 		Args:  cli.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			options.name = args[0]
+
+			f := cmd.Flag("ipv6")
+			if !f.Changed {
```

Comment:
```suggestion
			if !cmd.Flag("ipv6").Changed {
```

or (not sure which one is more performant, but probably won't matter much)

```suggestion
			if !cmd.Flags().Changed("ipv6") {
```

Reply:
Actually; would the reverse work? I see you have the `var ipv6 bool` added; that one could be set on the flag;



And this code could then do;

```go
if cmd.Flag("ipv6").Changed {
	options.ipv6 = &ipv6
}
```

Reply:
I chose `cmd.Flag("ipv6").Changed` instead of `cmd.Flags().Changed("ipv6")` as the former will trigger a panic if the flag is renamed / removed, whereas the latter will silently fail (ie. return `false`).